x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -61,9 +61,6 @@ const getSavedObjectTypes = (
       properties: {
```

Comment:
is it possible remove the SO mappings for `AGENT_EVENT_SAVED_OBJECT_TYPE`? will that break upgrade?

Reply:
I think it will be possible the SO should already not be migrated, but not sure if there other things to do, I will prefer if we do it early for 7.14 and have more testing around this, we can probably remove the `AGENT_ACTION_SAVED_OBJECT_TYPE`